Added a additonal areweanticheatyet checked (#76)
CachyOS/wiki: src/content/docs/configuration/gaming.mdx

:::

:::tip

> To check if your game is compatible or how does it run with Linux, visit [proton-db](https://www.protondb.com/) or [Areweanticheatyet (For Online Games and Online games not on steam)](areweanticheatyet.com).

Reviewer: ```suggestion
To check if your game is compatible or how does it run with Linux, visit [proton-db](https://www.protondb.com/) or [Are We Anti-Cheat Yet?](https://areweanticheatyet.com/).
```
